README.md

```diff
@@ -1,11 +1,36 @@
 # OpencC2Test
 
+OpenC2 is a standardized language for controlling cyber defense technologies. It allows different systems and components to communicate and coordinate with each other, regardless of the specific products or technologies they use. This enables automated and efficient management of cyber defenses.
```

Comment:
This describes the openc2 library instead of the project . 
Describe the project in the same way you have done in the UI changes